.github/workflows/test.yml

```diff
@@ -422,11 +435,15 @@ jobs:
           node-version: ${{ vars.NODE_VERSION }}
 
       - name: Dependencies
-        working-directory: ./
-        run: npm run ci:all
-
+        uses: ./.github/actions/retry
+        with:
+          command: npm run ci:all
+          working-directory: ./
```

Comment:
If you set working-directory as `./` as default in the module then you can remove all the working directory parameters.